mw-webapp/src/pages/mainPage/MainPage.tsx

```diff
@@ -1,3 +1,4 @@
+import {LoadingProvider} from "src/component/table/LoadingContext";
 import {Table} from "src/component/table/Table";
```

Comment:
Do you use LoadingProvider in this file? Sorry but dont see in code

Reply:
forget to delete import after cleaning up component